src/seedu/addressbook/commands/UndoCommand.java

```diff
@@ -0,0 +1,47 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.person.UniquePersonList;
+
+/**
+ * Clears the address book.
+ */
+public class UndoCommand extends Command {
+
+    public static final String COMMAND_WORD = "undo";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Reverts effect of previous command. "
+            + "Only applicable to add, delete commands.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Undo successful. Command reverted: ";
+    public static final String MESSAGE_INVALID = "Undo cannot be performed on this command: ";
+
+    private final String LAST_COMMAND;
+
+    public UndoCommand(String cmd){
+        this.LAST_COMMAND = cmd;
+    }
+
+    @Override
+    public CommandResult execute(){
+        String[] command = LAST_COMMAND.split(" ");
+        if(command[0].equals(AddCommand.COMMAND_WORD)){
+            try {
+              addressBook.removeLastPerson();
+            }
+            catch (UniquePersonList.PersonNotFoundException pnfe) {
```

Comment:
Catch statements should be on the same line as the closed brace of the previous try/catch closed brace.